builtin/logical/database/backend.go

```diff
@@ -140,11 +155,24 @@ func (b *databaseBackend) Role(s logical.Storage, roleName string) (*roleEntry,
 		return nil, nil
 	}
 
+	var upgradeCh upgradeCheck
```

Comment:
minor, I read this var as a channel. I think it might be easier to read if it was named `updateChk`.